includes/Services/SiteGenService.php

```diff
@@ -117,7 +117,7 @@ public static function instantiate_site_meta( $site_info, $identifier, $skip_cac
 			// Handle the error case by returning a WP_Error.
 			return new \WP_Error(
 				'nfd_onboarding_error',
-				__( 'Error generating site meta: ', 'wp-module-onboarding' ) . $identifier,
+				__( 'Error generating site meta: ', 'wp-module-onboarding' ) . $identifier . ' ' . $response['error'],
```

Comment:
The Sitegen class also returns an array sometimes for the same object `$response['error']` and sometimes a String, Need to find a better way else we might still be not able to surface the errors.

<img width="934" alt="Screenshot 2024-03-12 at 9 54 31 AM" src="https://github.com/newfold-labs/wp-module-onboarding-data/assets/48691514/092f767b-f26d-45ef-be53-d60ed6b54145">